src/java.desktop/macosx/native/libawt_lwawt/awt/CWrapper.m

```diff
@@ -343,6 +344,9 @@ static void initLevels()
     NSWindow *window = (NSWindow *)jlong_to_ptr(windowPtr);
     [ThreadUtilities performOnMainThreadWaiting:NO block:^(){
         [window setOpaque:(BOOL)opaque];
+        if ([window.contentView.layer isKindOfClass:[CAMetalLayer class]]) {
+            [window.contentView.layer setOpaque:(BOOL)opaque];
+        }
```

Comment:
This class "CWrapper" is intended to be a simple wrapper on top of cocoa methods, the changed method above is expected to call the "NSWindow#setOpaque" only. So you need to place "window.contentView.layer setOpaque" in some other place(native or java).

Reply:
Done